scripts/world_bank/boundaries/country_boundaries_mcf_generator.py

```diff
@@ -0,0 +1,202 @@
+)
+flags.DEFINE_string(
+    'mcf_dir', '',
+    'Dir to output generated MCF files too. If blank, a temp folder will be used.'
+)
+
+DOWNLOAD_URI = 'https://development-data-hub-s3-public.s3.amazonaws.com/ddhfiles/779551/wb_boundaries_geojson_highres.zip'
+TMP_PATH = '/tmp/wb_boundaries_geojson_highres'
+GEO_JSON_PATH = 'WB_Boundaries_GeoJSON_highres/WB_countries_Admin0.geojson'
+ISO_A3_CODES_TO_IMPORT = ['COD', 'ROU', 'TLS', 'AND', 'IMN', 'UMI', 'USA']
+EPS_LEVEL_MAP = {
+    0: 0,
+    0.01: 1,
+    0.03: 2,
+    0.05: 3
+}  # from scripts/us_census/geojsons_low_res/generate_mcf.py
+
+MCF_PATH = '/{MCF_OUT_FOLDER}/countries.dp{dp_level}.mcfgeojson.mcf'
+MCF_FORMAT_STR = "\n".join([
+    "Node: dcid:country/{country_code}", "typeOf: dcs:Country",
+    "{prop}: {coords_str}", "\n"
+])
+
+
+class CountryBoundariesMcfGenerator:
+
+    def __init__(self, download_dir, export_dir, mcf_dir):
+        self.download_dir = download_dir
+        self.should_download = False
+        if not self.download_dir:
+            self.download_dir = tempfile.mkdtemp(prefix="wb_download_")
+            self.should_download = True
+        self.export_dir = export_dir
+        if not export_dir:
+            self.export_dir = tempfile.mkdtemp(prefix="wb_export_")
+        self.mcf_dir = mcf_dir
+        if not mcf_dir:
+            self.mcf_dir = tempfile.mkdtemp(prefix="wb_mcf_")
+
+    def load_data(self):
+        """Download data from source, or use specified data from previous download.
+
+        Returns:
+          Geopandas dataframe with all source data loaded.
+        """
+        if self.should_download:
+            print(f'Downloading source data to {self.download_dir}')
+            r = requests.get(DOWNLOAD_URI, stream=True)
+            z = zipfile.ZipFile(io.BytesIO(r.content))
+            z.extractall(self.download_dir)
+        else:
+            print(f'Skipping download, reusing {self.download_dir}')
+        return gpd.read_file(os.path.join(self.download_dir, GEO_JSON_PATH))
+
+    def build_import_codes(self, all_countries):
+        """Builds mapping of column code to country codes to import.
+
+        Only countries with DCID of the form county/{code} are included.
+
+        Returns:
+          A dict mapping dataframe column names to iterable of values in that column to include in the import. e.g.:
+          {'WB_A3': ['FRA', 'NDL',...],
+          'ISO_A3': ['UMI', 'COD', ...]}
+        """
+        dc_all_countries = dc.query('''
+        SELECT ?place ?name WHERE {
+          ?place typeOf Country .
+          ?place name ?name
+        }
+        ''',
+                                    select=None)
+
+        def is_dc_country(iso):
+            dcid = f'country/{iso}'
+            matches = filter(lambda x: x['?place'] == dcid, dc_all_countries)
+            return len(list(matches)) > 0
+
+        wb_idx = all_countries['WB_A3'].apply(is_dc_country)
```

Comment:
maybe a comment somewhere as to what WB_A3 is?

Reply:
done, added that with the explanation of the extra map of country codes